pkg/test/resourcefixture/testdata/basic/sql/v1beta1/sqlinstance/postgresinstance/create.yaml

```diff
@@ -0,0 +1,55 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: sql.cnrm.cloud.google.com/v1beta1
+kind: SQLInstance
+metadata:
+  name: sqlinstance-sample-${uniqueId}
+spec:
+  databaseVersion: POSTGRES_14
+  region: europe-west4
+  settings:
+    availabilityType: REGIONAL
+    backupConfiguration:
+      backupRetentionSettings:
+        retainedBackups: 7
+        retentionUnit: COUNT
+      enabled: true
+      pointInTimeRecoveryEnabled: true
+      startTime: "05:00"
+      transactionLogRetentionDays: 3
+    databaseFlags:
+    - name: cloudsql.iam_authentication
+      value: "on"
+    - name: max_connections
+      value: "1000"
+    - name: max_worker_processes
+      value: "8" 
+    deletionProtectionEnabled: false
+    diskAutoresize: false
+    diskAutoresizeLimit: 0
+    diskSize: 100 
+    diskType: PD_SSD
+    insightsConfig:
+      queryInsightsEnabled: true
+      queryStringLength: 1024
+      recordApplicationTags: true
+      recordClientAddress: true
+    ipConfiguration:
+      ipv4Enabled: false
+      privateNetworkRef:
+        external: projects/${projectId}/global/networks/default
```

Comment:
`default` network is not guaranteed so you'll want to acquire/create it in the dependencies.yaml.

Context: https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/README.NewResourceFromTerraform.md#naming-conventions-special-variables-and-constants-in-testdata

Example: https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/v1.118.1/pkg/test/resourcefixture/testdata/basic/dns/v1beta1/dnsmanagedzone/dependencies.yaml

Reply:
Done, thank you so much for the example.  Super helpful

Reply:
Actually, it also needs to be updated here.

Example: https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/v1.118.1/pkg/test/resourcefixture/testdata/basic/dns/v1beta1/dnsmanagedzone/create.yaml#L29

Reply:
Comment again to make the response more noticeable (GitHub UI basically hide the response).

Reply:
Done, change the reference to k8s name